libraries/ESP8266WebServer/examples/FSBrowser/readme.md

```diff
@@ -0,0 +1,121 @@
+# FSBrowser readme
+
+## What is this sketch about ?
+
+This example is a FileSystem Browser for the ESP8266 using http requests and a html/javascript frontend, 
+working for both SPIFFS, LittleFS and SDFS.
```

Comment:
"work for all of" (both implies count of 2, here it's 3)

Reply:
Thanks. 
English is not my mother tongue, so all such tips are greatly appreciated.